docs/features.md

```diff
@@ -0,0 +1,43 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.DocFx addin.
+---
+The [Cake.Issues.DocFx addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from [DocFx] log files.
+
+  :::{.alert .alert-info}
+  [DocFx] can be run with [Cake.DocFx] addin.
+  :::
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.